### DISCUSSION TITLE
message when you do not have an answer

### DISCUSSION THREAD

**Discussion by meirinaldojunior** [hubotio/hubot#1603] [Q&A] [answered]
is it possible to set up an auto-reply message if there is no answer to the question?

**Comment by aflansburg** [comment, marked as answer]
There was mention of a catchAll() method in [issue #1394 ](https://github.com/hubotio/hubot/issues/1394), but I have not been able to get it to work on my Heroku-deployed Hubot.

**Comment by meirinaldojunior** [comment]
Ohh yeah!
I will read about.

thanks very much.
